docs/jwst/references_general/photom_reffile.inc

```diff
@@ -36,130 +36,218 @@ The FITS primary HDU does not contain a data array.
+|            |       | wavelength     | float     | array      | microns                |
++            +       +----------------+-----------+------------+------------------------+
+|            |       | relresponse    | float     | array      | unitless               |
 +            +       +----------------+-----------+------------+------------------------+
-|            |       | relresponse    | float     | 500        | unitless               |
+|            |       | reluncertainty | float     | array      | unitless               |
 +------------+-------+----------------+-----------+------------+------------------------+
 
-:Data model: `~jwst.datamodels.NircamPhotomModel`
+:Data model: `~jwst.datamodels.NrcImgPhotomModel`
 
 +------------+-------+----------------+-----------+------------+------------------------+
 | Instrument | Mode  | Column name    | Data type | Dimensions | Units                  |
 +============+=======+================+===========+============+========================+
-| NIRCam     | All   | filter         | string    | 12         | N/A                    |
+| NIRCam     | Image | filter         | string    | 12         | N/A                    |
 +            +       +----------------+-----------+------------+------------------------+
 |            |       | pupil          | string    | 12         | N/A                    |
 +            +       +----------------+-----------+------------+------------------------+
 |            |       | photmjsr       | float     | scalar     | MJy/steradian/(DN/sec) |
 +            +       +----------------+-----------+------------+------------------------+
 |            |       | uncertainty    | float     | scalar     | MJy/steradian/(DN/sec) |
++------------+-------+----------------+-----------+------------+------------------------+
+
+:Data model: `~jwst.datamodels.NrcWfssPhotomModel`
+
++------------+-------+----------------+-----------+------------+------------------------+
+| Instrument | Mode  | Column name    | Data type | Dimensions | Units                  |
++============+=======+================+===========+============+========================+
+| NIRCam     | WFSS  | filter         | string    | 12         | N/A                    |
++            +       +----------------+-----------+------------+------------------------+
+|            |       | pupil          | string    | 15         | N/A                    |
++            +       +----------------+-----------+------------+------------------------+
+|            |       | photmjsr       | float     | scalar     | MJy/steradian/(DN/sec) |
++            +       +----------------+-----------+------------+------------------------+
+|            |       | uncertainty    | float     | scalar     | MJy/steradian/(DN/sec) |
 +            +       +----------------+-----------+------------+------------------------+
 |            |       | order          | integer   | scalar     | N/A                    |
```

Comment:
A minor nit, but the order of the columns has changed in the actual schema. It's 1) filter, 2) pupil, 3) order, 4) photmjsr, 5) uncertainty, 6) nelem, 7) wavelength, 8) relresponse, 9) reluncertainty. So "order" needs to be moved up a couple rows in this table.